### PR TITLE
Misplaced paragraph:  talks about truncation in the memory per iterator section

### PR DESCRIPTION
**Pull request by vplauzon**

### CODE REVIEW

**Comment by PRMerger15** [comment]
@vplauzon : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: data-explorer/kusto/concepts/querylimits.md [hunk 1]
```diff
@@ -81,6 +81,12 @@ set truncationmaxrecords=1105;
 MyTable | where User=="Ploni"
 ```
 
+When considering removing these limits, first determine if
```
Comment (slavikn84) [inline data-explorer/kusto/concepts/querylimits.md:84]: I suggest moving the first sentence of this paragraph to the end of the first paragraph of the article, as it relates to all the limits. The rest of the paragraph should indeed be where you planted it.

**Comment by PRMerger12** [comment]
@vplauzon : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: data-explorer/kusto/concepts/querylimits.md (hunk 1 above)
Comment (vplauzon) [reply data-explorer/kusto/concepts/querylimits.md]: Agree & done.

**Comment by PRMerger8** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
Hi @orspod - Would you review the proposed revisions to your article? Thanks.

**Comment by orspod** [comment]
@vplauzon - thanks for your contribution. I am merging these changes now.

**Pull request merged**